### DISCUSSION TITLE
vue3官网广告条导致菜单显示不全

### DISCUSSION THREAD

**Discussion by cbw-cbw** [community/community#130134] [Programming Help]
### Select Topic Area

Bug

### Body

![屏幕截图 2024-06-28 145704](https://github.com/community/community/assets/112932875/df99f269-fbbe-42c4-a643-9e7868e6edaf)
![屏幕截图 2024-06-28 145821](https://github.com/community/community/assets/112932875/b295dd2e-9efd-4458-bdea-417cbddc3392)
![屏幕截图 2024-06-28 145855](https://github.com/community/community/assets/112932875/0353202e-e861-4b48-9836-89904e36232f)
是因为--vt-banner-height变量未定义，或者固定改为约80px

**Comment by ebndev** [comment]
Hi @cbw-cbw, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
